library/modules/DFSDL.cpp

```diff
@@ -5,6 +5,8 @@
 #include "Debug.h"
 #include "PluginManager.h"
 
+#include <SDL.h>
```

Comment:
If this is just for SDL_bool, looks like that's defined in `SDL_stdinc.h` if you want to narrow this